pkgs/by-name/ed/edopro/package.nix

```diff
@@ -0,0 +1,262 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, premake5
+, writeShellScriptBin
+, runCommandLocal
+, symlinkJoin
+, imagemagick
+, bzip2
+, curl
+, flac
+, fmt_9
+, freetype
+, libevent
+, libgit2
+, libGL
+, libGLU
+, libjpeg
+, libpng
+, libvorbis
+, libX11
+, libxkbcommon
+, libXxf86vm
+, lua5_3
+, nlohmann_json
+, noto-fonts-cjk-sans
+, openal
+, SDL2
+, sqlite
+, wayland
+, egl-wayland
+,
+}:
+let
+  edopro-version = "40.1.4";
+
+  # the assets repo has a font instead
+  font = "${noto-fonts-cjk-sans}/share/fonts/opentype/noto-cjk/NotoSansCJK-VF.otf.ttc";
```

Comment:
> optional: font discussion

Yes, the assets repo bundles a font (which is the same as this one), but this serves as a baked-in backup font in case the user e.g. deletes the font from their local directory. I don't think there's any harm in having this.